${EE_SIDE_PREFIX}_grasper_srv:
  ros__parameters:
    # The name of the controller,
    controller_name: ${EE_SIDE_PREFIX}_controller
    # Real joints that will be commanded 
    joint_names: 
      - ${EE_SIDE_PREFIX}_finger_joint

    # if the desired position differs by the actual position by
    # being > than 'max_position_error', the robot is grasping
    max_position_error: 0.15

    # Time for which the GripperGrasper analyzes the error (in s)
    timeout: 4.0

    # Publishing rate (of grasp state) - (in hz)
    rate: 1.0

    # Time for which to open/close the gripper
    opening_time: 0.5
    closing_time: 1.5

    # Open/closed joint value for pal-pro-gripper
    open_value: [0.0]
    close_value: [0.8]

